git-author.rb

```diff
@@ -14,6 +14,9 @@ def install
   def caveats; <<-EOS
     export GIT_TOGETHER_NO_SIGNOFF=1 to ~/.bash_profile
     to disable --signoff added by git-together commit
+
+    Please note the setup is incomplete, you must follow instructions in
```

Comment:
I will probably just replace the entire caveats (line 15 - 19) with:

`Please follow https://github.com/pivotal/git-author#setup for further setup instructions.`

Then, we can enhance the README.md overthere. Thoughts?

Reply:
Yeah that sounds good. I don't like trying to help people with the alias, since some people will use `bash_it` to set aliases, we just don't know if we should through it in `~/.bash_profile`.